data-prepper-plugins/kafka-plugins/src/test/resources/kafka-pipeline-bootstrap-servers-sasl-iam-role.yaml

```diff
@@ -0,0 +1,18 @@
+log-pipeline :
+  source:
+     kafka:
+        bootstrap_servers:
+          - "localhost:9092"
+        encryption:
+          type: "SSL"
+        authentication:
+          sasl:
+            aws_msk_iam: role
+        aws:
+          region: us-east-2
+          sts_role_arn: sts_role_arn
```

Comment:
Can you use something more unique to help ensure the assertion in the test is unique. Maybe `test_sasl_iam_sts_role`?